src/components/pages/account-page/components/group-tab/GroupTab.tsx

```diff
@@ -63,12 +62,16 @@ const GroupTab: FC = () => {
     return <NoGroupBlock />;
 
   if (!data || !user?.group || !user?.group.role) return null;
-  //TODO refactor this
   return (
-    <div>
-      <div>
-        <h4>Список групи {user.group.code}</h4>
-      </div>
+    <Box>
+      <Box>
+        <Typography variant="h4Bold">Список групи {user.group.code}</Typography>
```

Comment:
![image](https://github.com/fictadvisor/fictadvisor-web/assets/68693183/71a1b142-dc1f-4c3e-ac86-eb43bd7fbcfd)

change to h4